src/BlazorUI/Bit.BlazorUI/Components/Surfaces/Accordion/BitAccordion.scss

```
@@ -6,48 +6,158 @@
     font-weight: $tg-fw-regular;
     color: $clr-fg-pri;
     box-sizing: border-box;
-    font-size: $tg-fs-sm;
+    // The panel follows the size class of the accordion the way the header does, so that a Size means the
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)